server/routes/journeys/manage/visit/handlers/prisonerSelectHandler.ts

```diff
@@ -34,6 +56,9 @@ export default class PrisonerSelectHandler implements PageHandler {
         croNumber: prisoner.croNumber,
         prisonCode: prisoner.prisonId,
         prisonName: prisoner.prisonName,
+        restrictions,
+        alertsCount,
+        restrictionsCount,
       },
```

Comment:
This is good - just what we want - the restrictions list, and counts, store in the journey